src/components/organisms/AdminVenueView/components/Timing/Timing.tsx

```diff
@@ -58,10 +62,10 @@ export const Timing: React.FC<TimingProps> = ({
   return (
     <AdminPanel className="Timing">
       <AdminSidebar>
-        <AdminSidebarTitle>Plan your event</AdminSidebarTitle>
-        <TabFooter
-          {...tabNavigationProps}
-          handleVenueUpdate={handleVenueUpdate}
+        <AdminSidebarTitle>Plan your events</AdminSidebarTitle>
```

Comment:
```suggestion
        <AdminSidebarTitle>Plan your event</AdminSidebarTitle>
```

Reply:
based on the comment here https://github.com/sparkletown/internal-sparkle-issues/issues/1228#issue-1005704209